SkietbaanBE/SkietbaanBE/Controllers/UserController.cs

```diff
@@ -60,46 +60,35 @@ public async Task<IActionResult> AddUser([FromBody] User user)
                 return new BadRequestObjectResult("user cannot be null");
             }
         }
-        // PUT: api/User/
+        // PUT: api/User
         [HttpPut("{id}")]
         public async Task<IActionResult> UpdateUser(int id, [FromBody] User user)
         {
-            if (ModelState.IsValid)
```

Comment:
why are you remmoving modelstate.isvalid.  Your proposed new validation does not validate the entire object.

Reply:
i put it back